src/components/GroupingContainer/groupingContainer.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import {
+  Component,
+  Widget,
+  WidgetPropType,
+  InferWidgetProps
+} from "../Widget/widget";
+
+const GroupingContainerProps = {
+  name: PropTypes.string.isRequired,
+  children: PropTypes.node
+};
+
+// Generic display widget to put other things inside
+export const GroupingContainerComponent = (
+  props: InferWidgetProps<typeof GroupingContainerProps> & Component
+): JSX.Element => (
+  // Uses an inner margin for children similar to Phoebus
+  // This prevents the title being overwritten
+  // Could be changed or perhaps customisable as a prop
+  <fieldset
+    style={{
+      boxSizing: "border-box",
+      ...props.style
+    }}
+  >
+    <legend>{props.name}</legend>
+    <div style={{ position: "absolute", top: "20px", left: "20px" }}>
```

Comment:
How does this affect relatively positioned children inside this div? Perhaps margin could be used to provide the same effect?

Reply:
Just making it a relatively placed div with 100% height and width seems to put it in a pretty good place (slightly different from xml but sensible), guess HTML people have already thought of this and sorted it for us!